jdi-light/src/main/java/com/epam/jdi/light/actions/ActionHelper.java

```diff
@@ -61,19 +61,29 @@
  * Email: roman.iovlev.jdi@gmail.com; Skype: roman.iovlev
  */
 public class ActionHelper {
+    private static final JDISettings jdiSettings = getJDISettings();
+    private static final WebSettings webSettings = getWebSettings();
+
     static String getTemplate(LogLevels level) {
-        if (LOGS.logInfoDetails != null) {
-            switch (LOGS.logInfoDetails) {
-                case NONE: return "{action}";
-                case NAME: return "{action} ({name})";
-                case LOCATOR: return "{action} ({locator})";
-                case CONTEXT: return "{action} ({context})";
-                case ELEMENT: return "{action} ({element})";
+        if (jdiSettings.LOGS.logInfoDetails != null) {
+            switch (jdiSettings.LOGS.logInfoDetails) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Switch statements should have a default label](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5482899)